packages/framework/ai-collab/src/explicit-strategy/utils.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Subset of Map interface.
+ *
+ * @remarks - originally from tree/src/utils.ts
```

Comment:
Nit, also applies to the functions below.

Note for future reference: we should consider moving these into client-utils for reuse.
```suggestion
 * @remarks - originally from tree/src/util/utils.ts
```

Reply:
`@remarks` tags also shouldn't be followed by an `-`.
```suggestion
 * @remarks Originally from `tree/src/util/utils.ts`
```

Reply:
Updated in commit bac309434408ff1fd2adfbc0dcacf19c89f40ee8